collect_app/src/androidTest/java/org/odk/collect/android/InstancesDaoTest.java

```diff
@@ -0,0 +1,248 @@
+/*
```

Comment:
We don't have a lot of tests now but we will soon 😊 so I think the test package structure should match the code package structure. This should be in android/dao/